### ISSUE TITLE
Create a table or tables of NBA player "common" info (age, debut, height, weight, wingspan, vertical, etc.)

### ISSUE THREAD

**Issue by jimtheflash** [jimtheflash/gambling_stuff#13]
Updating: please create a table or tables of NBA player common info, i.e. everything from this [endpoint](https://github.com/swar/nba_api/blob/master/docs/nba_api/stats/endpoints/commonplayerinfo.md), for all players in gamelogs.

**Comment by jimtheflash** [comment]
Updated to reflect the correct endpoint and scope of players.

**Comment by jimtheflash** [comment]
Creating a separate issue for the job to update.

**Comment by johnlovesdata** [comment]
Here are the fields available in CommonPlayerInfo. This appears to be an updating data source, like gamelog, and not a point in time data source. So every player is represented once and only once. This does appear to get updated with each new season (which is why 2020 is listed for every current player). So I imagine we'd want a way to fully refresh for each new season. 

```
PERSON_ID
FIRST_NAME
LAST_NAME
DISPLAY_FIRST_LAST
DISPLAY_LAST_COMMA_FIRST
DISPLAY_FI_LAST
PLAYER_SLUG
BIRTHDATE
SCHOOL
COUNTRY
LAST_AFFILIATION
HEIGHT
WEIGHT
SEASON_EXP
JERSEY
POSITION
ROSTERSTATUS
GAMES_PLAYED_CURRENT_SEASON_FLAG
TEAM_ID
TEAM_NAME
TEAM_ABBREVIATION
TEAM_CODE
TEAM_CITY
PLAYERCODE
FROM_YEAR
TO_YEAR
DLEAGUE_FLAG
NBA_FLAG
GAMES_PLAYED_FLAG
DRAFT_YEAR
DRAFT_ROUND
DRAFT_NUMBER
```
